_topic_maps/_topic_map.yml

```diff
@@ -1349,18 +1349,10 @@ Topics:
       File: aws-load-balancer-operator-release-notes
     - Name: AWS Load Balancer Operator in OpenShift Container Platform
       File: understanding-aws-load-balancer-operator
+    - Name: Preparing an AWS STS cluster for the AWS Load Balancer Operator
+      File: installing-albo-sts-cluster
     - Name: Installing the AWS Load Balancer Operator
       File: install-aws-load-balancer-operator
-    - Name: Installing the AWS Load Balancer Operator on a cluster that uses AWS STS
-      File: installing-albo-sts-cluster
-    - Name: Creating an instance of the AWS Load Balancer Controller
-      File: create-instance-aws-load-balancer-controller
-    - Name: Serving multiple ingress resources through a single AWS Load Balancer
-      File: multiple-ingress-through-single-alb
-    - Name: Adding TLS termination
-      File: add-tls-termination
-    - Name: Configuring cluster-wide proxy
-      File: configure-egress-proxy-aws-load-balancer-operator
```

Comment:
For each of these removed removed assemblies, I'd suggest doing a SHIFT+CTRL+F (a cross-repo search) locally for each of those strings (e.g., `installing-albo-sts-cluster`, `create-instance-aws-load-balancer-controller`, and so on).

There appear to be metadata remaining that reference these files (like the "Module included in the following assemblies:" at the tops of modules), and I'm not sure if they should be touched-up to reference a new location (if the module is in fact still in use), or if the references are in files that should just be fully removed at this point (if the content is no longer are intended to appear anywhere).

Reply:
Modules in this particular subset of documentation are so niche that they shouldn't be re-used anywhere, and most of them were the only modules in the assembly they were used in, so removing the assembly altogether will be the move. Mostly just have to look out for any references to the assemblies.